config/config_test.go

```diff
@@ -19,51 +19,76 @@ func (m *mockConfigDirectoryFetcher) GetUserConfigDir() (string, error) {
 	return m.dir, nil
 }
 
-func prepareSeshConfig(t *testing.T) string {
+type MockSeshConfig struct {
+	Name     string
+	Contents string
+	Imports  string
+}
+
+func prepareSeshConfig(t *testing.T, mockSeshConfigs []MockSeshConfig) string {
```

Comment:
inject a MockSeshConfig list to enable this function to be used multiple times for many tests.

This allows the caller to create their own MockSeshConfig and get a mock config back.